assets/sass/components/_filter.scss

```
@@ -0,0 +1,9 @@
+/**
+ * Local style overrides for the MoJ Filter component
+ * https://design-patterns.service.justice.gov.uk/components/filter/
+ */
+
+.moj-filter__header {
```

```suggestion
.moj-filter__header--light-grey {
```

Could we use BEM modifer class names instead of modifying the component class directly? Its easier to identify that this is our own implementation although if we are modifying components we should really try to push the changes back to moj frontend else moj will end up have 1000 slightly different component instead of a single reusable and consistent component.

---

That would be ideal, but unfortunately the macro doesn't enable the use of any modifier classes, hence the extra rules added to the source class (https://github.com/ministryofjustice/moj-frontend/blob/main/src/moj/components/filter/template.njk#L3). I'm open to any other suggestion, I don't like it much myself this way!

---

😬 not much you can customise with that component. 

Could we make a contribution? we could add custom classes to the component itself or to the header something like how govuk do it https://github.com/alphagov/govuk-frontend/blob/main/packages/govuk-frontend/src/govuk/components/back-link/template.njk#L3

its a pitty the head/body parts of the component where not split out as a micro components themselves where you could pass in custom css class names or attributes to the different part.

---

https://github.com/ministryofjustice/moj-frontend/issues/611